README.md

```diff
@@ -1,7 +1,35 @@
-# Settings View package [![Build Status](https://travis-ci.org/atom/settings-view.svg?branch=master)](https://travis-ci.org/atom/settings-view)
+# Settings View [![Build Status](https://travis-ci.org/atom/settings-view.svg?branch=master)](https://travis-ci.org/atom/settings-view)
 
-Edit config settings, install packages, and change themes in Atom.
+Edit config settings, install packages, and change themes all from within Atom.
```

Comment:
What do you think about adding two more words just to make things a bit clearer?

> Edit core config settings, install and configure packages, and change themes all from within Atom.